app/views/directives/lifecycle-hook.html

```diff
@@ -0,0 +1,151 @@
+          <div class="help-block">
+            Environment variables to supply to the hook pod's container.
+          </div>
+        </div>
+
+        <div class="form-group">
+        <label>Volumes</label>
+        <ui-select multiple placeholder="Select volume" ng-model="hookParams.execNewPod.volumes" ng-disabled="view.isDisabled">
+          <ui-select-match>{{$item}}</ui-select-match>
+          <ui-select-choices repeat="volume in availableVolumes | filter : $select.search">
+            <div ng-bind-html="volume | highlight : $select.search"></div>
+          </ui-select-choices>
+        </ui-select>
+        <div class="help-block">
+          List of named volumes to copy to the hook pod.
+        </div>
+        </div>
+      </div>
+
+      <div ng-if="action.type === 'tagImages'">
+<!--         <div class="form-group">
+          <label class="required">Container Name</label>
+          <ui-select ng-model="hookParams.tagImages[0].containerName" ng-disabled="view.isDisabled" required>
+            <ui-select-match>{{$select.selected}}</ui-select-match>
+            <ui-select-choices repeat="container in (availableContainers | filter : $select.search)">
+              <div ng-bind-html="container | highlight : $select.search"></div>
+            </ui-select-choices>
+          </ui-select>
+          <div class="help-block">
+            Use the image for this container as the source of the tag.
+          </div>
+        </div>
+
+        <div class="form-group">
+          <label class="required">Tag As</label>
+          <istag-select model="istagHook" allow-custom-tag="true" select-disabled="view.isDisabled"></istag-select>
+        </div> -->
+
+
+        <div ng-repeat="tagImage in hookParams.tagImages">
+          <div ng-if="$first">
+            <div class="form-group">
+              <label class="required">Container Name</label>
+              <ui-select ng-model="tagImage.containerName" ng-disabled="view.isDisabled" required>
+                <ui-select-match>{{$select.selected}}</ui-select-match>
+                <ui-select-choices repeat="container in (availableContainers | filter : $select.search)">
+                  <div ng-bind-html="container | highlight : $select.search"></div>
+                </ui-select-choices>
+              </ui-select>
+              <div class="help-block">
+                Use the image for this container as the source of the tag.
+              </div>
+            </div>
+            <div class="form-group">
+              <label class="required">Tag As</label>
+              <istag-select model="istagHook" allow-custom-tag="true" select-disabled="view.isDisabled"></istag-select>
+            </div>
+            <p ng-if="hookParams.tagImages.length > 1">
+              <span class="pficon pficon-info" aria-hidden="true"></span>
+              More then one image tags are defined in this deployment configuration. Only the first one is editable, below is displayed the rest.
```

Comment:
It's weird that the first one only is editable if there is more than one. Either all image tags should be read-only (when there are multiple) or we should just try to support editing all of them.